Gems/PhysX/Code/Source/EditorRigidBodyComponent.cpp

```diff
@@ -126,6 +127,30 @@ namespace PhysX
             &AzToolsFramework::EditorRequests::OpenViewPane, LyViewPane::PhysXConfigurationEditor);
     }
 
+    static int OnEditButtonClicked(int index)
```

Comment:
question: This change looks good but again was just curious if we can have this return a `bool`? Do we need the additional `-1` to detect it failed? I'm just wondering if we could simplify this or of there's a reason we have to use an `int`. Otherwise looks good. Thanks!

Reply:
Changed to bool :)

Reply:
Awesome thanks!